examples/headless/Cargo.toml

```diff
@@ -1,23 +1,19 @@
 [package]
 name = "headless"
-description = "An example showing how to use `vello` to create raster images"
-publish = false
-
-version.workspace = true
-license.workspace = true
+description = "An example showing how to use `vello` to create raster images."
```

Comment:
I *think* this should use Vello (with a capital letter) outside of graves. I'm not even sure that the crate description is markdown rendered.

Reply:
Good call!